docroot/themes/custom/twentyeighteen/sass/components/hero/hero.twig

```diff
@@ -0,0 +1,13 @@
+<div class= "hero">
+  <div class= "hero__background-img">
+    <img src="../images/wide_home-hero.jpg"/>
+  </div>
+  <div class= "hero__logo">
```

Comment:
The only thing I would change here markup wise is have the logo and the button under one wrapper. Maybe hero__details, hero__detatils-logo, hero__details-callout...mostly so with absolute positioning you don't have to redo several elements for different breakpoints.